MAUI/LinearGauge/content-marker-pointer.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: post
+title: Content Marker Pointer in .NET MAUI Linear Gauge control | Syncfusion
+description: Learn here all about adding and customizing Content Marker Pointer of Syncfusion .NET MAUI Linear Gauge (SfLinearGauge) control and more.
+platform: maui
+control: SfLinearGauge
+documentation: ug
+---
+
+# Content Marker Pointer in .NET MAUI Linear Gauge (SfLinearGauge)
+
+The [`LinearContentPointer`]() in [`SfLinearGauge`]() allows to use any .NET MAUI content as marker pointer. The following code sample uses a [`Image`]() as marker pointer.
```

Comment:
The [`LinearContentPointer`]() in [`SfLinearGauge`]() allows you to use any .NET MAUI content as a marker pointer. The following code sample uses a [`Image`]() as a marker pointer.